package.json

```diff
@@ -2,11 +2,16 @@
   "name": "react-formbuilder",
   "version": "0.0.1",
   "description": "A relatively easy to use form builder (single or multipage), with error reporting and lots of room for customization ",
-  "main": "index.js",
+  "main": "dist/react-formbuilder.js",
   "directories": {
     "test": "test"
```

Comment:
we can get rid of it - it's not actually used for anything by the current version of npm